app/.gitignore

```diff
@@ -0,0 +1,41 @@
+# Miscellaneous
```

Comment:
Could these items be moved to the existing .gitignore and still work?

Reply:
I'll have to try this and see if it'll work

Reply:
Using our existing .gitignore did not work.